libcircle.spec

```diff
@@ -66,25 +68,26 @@ A simple interface for processing workloads using an automatically
 distributed global queue.
 
 This package contains development headers and libraries for
-MPICH ibcircle
+MPICH libcircle
 
 %prep
-%setup -q -n %{name}-%{myversion}
+%setup -q
+./autogen.sh
 
 %build
 mkdir openmpi mpich
-%global dconfigure %(printf %%s '%configure' | sed 's!\./configure!../configure!g')
+%global _configure ../configure
```

Comment:
Is this used anywhere?

Reply:
Don't know.  That is from the original fedora spec file for 0.3 tag.

Reply:
What is the link to the spec you are using?

Reply:
I see what they are doing.